server/controller/accountsMangment/captains/editCaptain.js

```diff
@@ -14,7 +16,12 @@ const updateCaptain = (req, res) => {
     const { id } = req.params;
     const extName = path.extname(file.name).slice(1);
     const newName = `${new Date().getTime()}id_photo.${extName}`;
-    if (['png', 'jpg', 'jpeg'].some(ext => ext === extName)) {
+    const { error } = joi.validate({ ...req.body }, editCaptainSchema);
+    if (error) {
+      res.status(404).send({
+        error: 'captain dose not edit',
```

Comment:
if their error in validation the status code is `400`  and the message is `Bad Request`